### PR TITLE
Revise content on Slurm partitions

### PR DESCRIPTION
**Pull request by msainio**
## Proposed changes

Reorganize page on available batch job partitions and improve instructions for using multi-instance GPUs.
Revise page on interactive usage.

[batch-job-partitions.md]
[interactive-usage.md]

## Checklist before requesting a review

- [x] I have followed the instructions in the [Contributing](https://github.com/CSCfi/csc-user-guide/blob/master/CONTRIBUTING.md) and [Styleguide](https://github.com/CSCfi/csc-user-guide/blob/master/STYLEGUIDE.md) documents.
- [x] My pull request passes the automated tests. If not, visit [pull requests at Travis CI](https://app.travis-ci.com/github/CSCfi/csc-user-guide/pull_requests) and have a look at the job log.
- [x] I have included a link to the appropriate [preview page](https://csc-guide-preview.2.rahtiapp.fi/origin/) (select your branch from the list).

[batch-job-partitions.md]: https://csc-guide-preview.rahtiapp.fi/origin/revise-partitions-page/computing/running/batch-job-partitions/
[interactive-usage.md]: https://csc-guide-preview.rahtiapp.fi/origin/revise-partitions-page/computing/running/interactive-usage/

### CODE REVIEW

File: docs/computing/running/batch-job-partitions.md [hunk 1]
```diff
@@ -1,100 +1,173 @@
...
-
-```
-sinfo
+On CSC supercomputers, programs are run by submitting them to partitions,
+which are logical sets of nodes managed by the SLURM workload manager.
+This page lists the available SLURM partitions on the Puhti and Mahti
+supercomputers, as well as explains their intended uses. Below are the general
+guidelines for using the SLURM partitions on our systems:
+
+1. **Use the `test` and `gputest` partitions for developing your code.** These
```
Comment (rkronberg) [inline docs/computing/running/batch-job-partitions.md:9]:
```suggestion
1. **Use the `test` and `gputest` partitions for testing your code, not production.** These
```

**Comment by msainio** [comment]
What is our preferred term for MIGs?

**Review by jlento — APPROVED** [review]
Looks clean!

**Comment by msainio** [comment]
PR pending update to `sinteractive` command

**Comment by msainio** [comment]
Marking as ready since`sinteractive` has been updated

**Comment by rkronberg** [comment]
> Marking as ready since`sinteractive` has been updated

Adding review in just a bit!

**Review by rkronberg — APPROVED** [review]
Great work!

**Pull request merged**